### PR TITLE
Small batch of QoL changes and requests

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

- Adds a cat eared alt to the Reaver SCAF suit, long requested by Trilby but I never got around to do it
- Reaver SCAF suit is 20% faster than a normal SCAF (incentive to loot it over the SCAF at Ironhead)
- GP thermals now cost platinum, and are more battery efficient than normal thermals, to justify researching its node (previously they were literally just reskinned thermals, nothing more)
- Refactors Greyson multitool into a PROPER multitool, so you can hack dispensers with it
- Guild nightvisions made 10% more battery efficient, they should last 10 more seconds instead of just 5 more seconds than a normal NV
- Scout heavy boltgun should now be able to take stripper clips for ease of reloading
- Deletes many duplicate sprites
- Touchup of fancy watches to make them fancier
- Replaces Makeshift Voidsuit Helmet on prospector/salvager/foreman lockers (on Scavenger Armor chance roll) with the Scavenger Helmet, more thematically fitting
- Miscellaneous typo fixes

<hr>

## Changelog
:cl:
tweak: Replaced Makeshift Voidsuit Helmet on Foreman/Prospector/Salvager lockers with the Scavenger Helmet on the chance that it rolls with the Scavenger Armor.
tweak: GP Thermals now cost platinum to make, and are more battery efficient than the average thermal goggles
balance: Guild NV goggles made 10% more battery efficient, as they were merely lasting 5 more seconds than the average NV goggles. Sorry guild, I'll learn to code better and make them toggleable mesons/nv on demand...
balance: Reaver SCAF has less slowdown than the normal SCAF, because red makes it go FASTA
fix: Scout should be able to take stripper clips now
imageadd: Adds the long requested Reaver SCAF cat eared variant, as every other SCAF had it before
imagedel: deleted some duplicated icons
spellcheck: fixed a few typos
refactor: Greyson multitools are now proper multitools able to hack dispensers
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/tools/multitool.dm [hunk 1]
```diff
@@ -40,7 +40,6 @@
 	degradation = 2
 	workspeed = 1.6
 	max_upgrades = 5 //UNLIMITED MODS!
-	suitable_cell = /obj/item/cell/medium
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/multitool.dm:43]: why are you making it take a small cell?

Comment (ThePainkiller) [reply code/game/objects/items/weapons/tools/multitool.dm:43]: Trilby, dear, I made the greyson multitool a children of the ADVANCED multitool, it should inherit the medium cell from it

Comment (ThePainkiller) [reply code/game/objects/items/weapons/tools/multitool.dm:43]: Unless the whole block of Greyson multitool code has to be written BELOW the advanced multitool one?

**Comment by ThePainkiller** [comment]
```
apology for poor english

when were you when dilf bot dies

i was sat at home drinking coffer when trilby ring

'diffbot is dead'

'no'

and you?????????????
```

**Pull request merged**